act-rules-format.bs

```diff
@@ -699,6 +714,128 @@ This section is *non-normative*.
 }
 ```
 
+**Example 5:** Assumed 'Context' for this section
+
+```javascript
+{
+  "@vocab": "http://www.w3.org/ns/earl#",
+
+  "cnt": "http://www.w3.org/2011/content#",
+  "dct": "http://purl.org/dc/terms/",
+  "earl": "http://www.w3.org/ns/earl#",
+  "foaf": "http://xmlns.com/foaf/spec/",
+  "htp": "http://www.w3.org/2011/http#",
+  "ptr": "https://www.w3.org/2009/pointers#",
+  "schema": "http://schema.org/",
+  "xsd": "https://www.w3.org/2001/XMLSchema#",
+
+  "WCAG20": "https://www.w3.org/TR/WCAG20#",
+  "WCAG21": "https://www.w3.org/TR/WCAG21#",
+  "ACT-CG": "https://act-rules.github.io/",
+
+  "WebSite": "schema:WebSite",
+  "WebPage": "schema:WebPage",
+
+  "title": "dct:title",
+  "description": "dct:description",
+  "date": "dct:date",
+  "hasVersion": "dct:hasVesion", 
+  "isPartOf": "dct:isPartOf", 
+  "hasPart": "dct:hasPart",
+  "source": "dct:source",
+
+  "Agent": "foaf:term_Agent",
+  "Person": "foaf:term_Person",
+  "Organization": "foaf:term_Organization",
+  "Group": "foaf:term_Group",
+  "Document": "foaf:term_Document",
+  "name": "foaf:term_name",
+  "firstName": "foaf:term_firstName",
+  "surname": "foaf:term_surname",
+  "mbox": "foaf:term_mbox",
+  "mbox_sha1sum": "foaf:term_mbox_sha1sum",
+  "member": "foaf:term_member",
+  "homepage": "foaf:term_homepage",
+
+  "Assertion": "earl:Assertion",
```

Comment:
I think you should remove all of these `earl:` classes. Because you set `@vocab`, they already resolve to earl by default. Also, giving a class a type doesn't really make sense. Only properties can have a value, and so only properties can have a default type. Classes are types. It looks like you're trying to express inheritance in this, but that's not what `@type` is for.

Reply:
Done.